filecoin-proofs/src/api/seal.rs

```diff
@@ -352,6 +352,8 @@ pub fn seal_commit_phase1<T: AsRef<Path>, Tree: 'static + MerkleTreeTrait>(
     prover_id: ProverId,
     sector_id: SectorId,
     ticket: Ticket,
+    // Note: when using NI-PoRep the PoRep challenge generation seed is ignored, thus any value can
+    // be passed in here for `seed`.
```

Comment:
Just a note (feel free to ignore). I know we don't really use API docs here (their building is even broken). Though it could make sense to put it into the top of the function (or another note there) as a doc comment, in case we ever properly generate those.